src/css/card.css

```diff
@@ -0,0 +1,98 @@
+.card {
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  background-color: var(--white-color);
+  border: var(--card-border);
+  border-radius: 5px;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  display: block;
+  margin: 0 3px 40px;
```

Comment:
`3px` is strange value, just use auto for positioning in the center `margin: 0 auto 40px;`